regression/cbmc-output-file/outfile/z3-no-match.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+test.c
+--incremental-smt2-solver 'z3 --smt2 -in' --slice-formula --outfile %out-file-name%
+Passing problem to incremental SMT2 solving via SMT2 incremental dry-run
+Output file does not match.
+^EXIT=1$
+^SIGNAL=0$
+--
+Test to check that the --outfile argument is used correctly and that the output file matches the
+rules defined in test-outfile-rules.txt
```

Comment:
:pick: This file is missing a trailing newline character.

Reply:
Added.